frontend/src/pages/CoursesList/styled.ts

```diff
@@ -1,25 +1,20 @@
+import { Grid } from '@mui/material';
+
 import styled from 'styled-components';
 
 import theme from 'themeSettings';
 import Button from 'components/Button';
 
-export const PageContainer = styled('div')({
-  [theme.breakpoints.up('lg')]: {
+export const PageContainer = styled(Grid)({
+  [theme.breakpoints.down('xl')]: {
     maxHeight: '100vh',
     maxWidth: '100%',
     overflowY: 'scroll',
     margin: '0px !important',
-    marginTop: '25px !important',
-  },
```

Comment:
Do we need Grid here? Without grid all elements were already in right order and positions

Reply:
Grid is easier and simply for work in future . Grid "from box" have props for responsive design  and each grid element has a lot of settings to position an element. With grid it's easier to make changes - 12colums vs 3 and huge custom feature. If we will have to change our Courses page its will be too easier to do with grid.